### PR TITLE
Apply copy editor changes to invalidation.md

### PR DESCRIPTION
**Pull request by chrishtr**

### CODE REVIEW

File: book/invalidation.md [hunk 1]
```diff
@@ -29,26 +29,15 @@ a few frames' delay is distracting. But editing changes the HTML tree
 and therefore the layout tree. Rebuilding the layout tree from
 scratch, which our browser currently does, can be very slow on
 complex pages. Try, for example, loading [this
-chapter](invalidation.md) in our browser and typing into this
-input box:
+chapter](https://browser.engineering/invalidation.html) in our browser and typing into the input box
+that appears after this paragraph...You'll find that it is *much* too
+slow---1.7 seconds just in `render` (see Figure 1)!
+^[Trace [here](https://browser.engineering/examples/example16-input-no-optimizations.trace).]
```
Comment (pavpanchekha) [inline book/invalidation.md:35]: Can you confirm that there's no space between the `!` and the footnote in the PDF build?

File: book/invalidation.md [hunk 2]
```diff
@@ -162,7 +151,8 @@ class InputLayout(EmbedLayout):
             cmds.append(DrawCursor(self, self.font.measureText(text)))
 ```
 
-You can now edit the examples on this page in your browser---but
+You can now edit the examples on
+[this page](https://browser.engineering/invalidation.html) in your browser---but
```
Comment (pavpanchekha) [inline book/invalidation.md:155]: In other places the copyeditor suggests "the web version of this chapter". Did he not, here? Or you decided against?

File: book/invalidation.md [hunk 3]
```diff
@@ -1530,7 +1521,7 @@ establishes a dependency for one layout field upon another. I like to
 think of each `f` as being scoped to its field's computation.
 
 We also need to compute the `x` position of a `TextLayout`. That can
-use the previous sibling's font, *x* position, and width:
+use the previous sibling's font, `x` position, and width:
```
Comment (pavpanchekha) [inline book/invalidation.md:1524]: Same comment as in the other PR, that I think italics here is correct. (Or, code font is arguable, but only if we make `font` and `width` also code font.)

File: book/invalidation.md [hunk 4]
```diff
@@ -1878,23 +1869,17 @@ element's `children` (which we have to do, to incorporate the new
...
 we wrapped onto more lines).
 
-::: {.web-only}
-
 Editing should also now feel snappier---about
 0.6 seconds instead of the original 1.7. Better, but still not good:
-^[Trace [here](examples/example16-input-reuse-layout-tree.trace).]
-
-:::
+^[Trace [here](http://browser.engineering/examples/example16-input-reuse-layout-tree.trace).]
```
Comment (pavpanchekha) [inline book/invalidation.md:1874]: Same question about no space between `:` and the footnote.

File: book/invalidation.md [hunk 5]
```diff
@@ -2769,26 +2744,24 @@ invalidation properly.
...
 whose styles are unchanged.
 
-16-5 *Resizing the browser*. Perhaps, back in [Chapter
-2](graphics.md#exercises), you implemented support for resizing the
-browser. (And most likely, you dropped support for it when we switched
-to SDL.) Reimplement support for resizing your browser; you'll need to
-pass the `SDL_WINDOW_RESIZABLE` flag to `SDL_CreateWindow` and listen
-for `SDL_WINDOWEVENT_RESIZED` events. Make sure invalidation works:
-resizing the window show resize the page. How much does invalidation
+16-5 *Resizing the browser*. Perhaps, in Exericse 2-3 you implemented
```
Comment (pavpanchekha) [inline book/invalidation.md:2754]:
```suggestion
16-5 *Resizing the browser*. Perhaps, in Exericse 2-3, you implemented
```

**Review by pavpanchekha — CHANGES_REQUESTED** [review]

File: book/invalidation.md (hunk 1 above)
Comment (chrishtr) [reply book/invalidation.md]: It was not- fixed just now.

File: book/invalidation.md (hunk 2 above)
Comment (chrishtr) [reply book/invalidation.md:155]: They recommended what I put here.

**Comment by chrishtr** [comment]
Please re-review.

**Review by pavpanchekha — APPROVED** [review]

**Pull request merged**